Readme.md

```diff
@@ -237,6 +237,15 @@ full_path=%month/%year/%album|%location|%"Beats me"
 
 What this asks me to do is to name the last folder the same as the album I find in EXIF. If I don't find an album in EXIF then I should use the location. If there's no GPS in the EXIf then I should name the last folder `Beats me`.
 
+#### Custom Default Location
+
+If I wasn't able to find a location for your photo I will use the default location "Unknown Location". This can be changed via your `config.ini` file
+
+```
+[UnknownLocation]
+folder_name=Ukendt placering
```

Comment:
Can we reuse `[Directory]` and name the setting `default_folder_name`?

Reply:
but if I use the `[Directory]` wont that trigger the custom path section of the code? I want to be able to use the default path and only changing the fallback directory